packages/tokenbank_v1/contracts/modules/transfers/QuotaTransfers.sol

```diff
@@ -112,6 +112,29 @@ contract QuotaTransfers is TransferModule
         }
     }
 
+    function transferTokensFullBalance(
+        address            wallet,
+        address[] calldata tokens,
+        address            to,
+        bytes calldata     logdata
+        )
+        external
+        nonReentrant
+        onlyFromWalletOwner(wallet)
+        onlyWhenWalletUnlocked(wallet)
+    {
+        bool allowed = whitelistStore.isWhitelisted(wallet, to);
+
+        if (allowed) {
```

Comment:
Maybe this shouldn't silently do nothing, and just throw instead?

Reply:
made it throw.